internal/pkg/composable/providers/kubernetes/pod.go

```diff
@@ -492,16 +503,22 @@ func generateContainerData(
 	}
 }
 
-func getHintsMapping(k8sMapping map[string]interface{}, logger *logp.Logger, prefix string, cID string) mapstr.M {
+func getHintsMapping(k8sMapping map[string]interface{}, logger *logp.Logger, prefix string, cID string) (mapstr.M, []mapstr.M) {
 	hintsMapping := mapstr.M{}
+	processorMapping := []mapstr.M{}
+
 	if ann, ok := k8sMapping["annotations"]; ok {
 		annotations, _ := ann.(mapstr.M)
 		hints := utils.GenerateHints(annotations, "", prefix)
 		if len(hints) > 0 {
 			logger.Debugf("Extracted hints are :%v", hints)
 			hintsMapping = GenerateHintsMapping(hints, k8sMapping, logger, cID)
 			logger.Debugf("Generated hints mappings are :%v", hintsMapping)
+
+			processorMapping = utils.GetConfigs(annotations, prefix, "hints/processors")
```

Comment:
How about defining the `hints/processors` as a const for the package?

Reply:
Thinking what we will need to do here in case we want to support multiple containers. 
So the annotation would be `co.elastic.hints.<container>/processors` right? I will need to test this

Reply:
Check how it's done in Filebeat: https://www.elastic.co/guide/en/beats/filebeat/7.17/configuration-autodiscover-hints.html#_multiple_containers

Reply:
Constant added https://github.com/elastic/elastic-agent/pull/3107/files#diff-ed933ad9cf09eb9ca797c5c89b5e8991624cbe5301d88f996d9bd947f4e3ea16R27